io/bmp/Bmp.ml

```diff
@@ -0,0 +1,30 @@
+open Bigarray
+
+type data = (int32, int32_elt, c_layout) Array1.t
+
+module IO : Odiff.ImageIO.ImageIO = struct
```

Comment:
Since you already have the `Bmp.mli` interface file, the annotations here are redundant...

Reply:
This just helps local editor to give autocomplete in this module instantly, when the type defined in separate module error often is not propagated correctly.